Bio/Restriction/__init__.py

```diff
@@ -11,6 +11,8 @@
 
 Examples
 --------
+.. code-block:: python
```

Comment:
Does this help? I thought it was redundant with doctest markup.

Reply:
it is necessary to avoid badly formatted output, like this:
<img width="739" alt="Screen Shot 2020-07-12 at 10 15 49" src="https://user-images.githubusercontent.com/710117/87253110-28380d80-c43e-11ea-9504-6b8a79ea896d.png">

The blank line before BmeDI avoids the whole block to be completely formatted. This is fixed by adding `.. code-block:: python`:
<img width="799" alt="Screen Shot 2020-07-12 at 12 51 22" src="https://user-images.githubusercontent.com/710117/87253152-659c9b00-c43e-11ea-970c-062cce5e2249.png">

Reply:
Ah - good reason to avoid blank lines within output. Thanks!